test/trento/personal_access_tokens_test.exs

```diff
@@ -283,6 +284,70 @@ defmodule Trento.PersonalAccessTokensTest do
     end
   end
 
+  describe "validating and introspecting a Personal Access Token" do
+    test "should return an error when validating and introspecting a non existent PAT" do
+      # invalid jti/user_id combination
+      assert {:error, :invalid_token} =
+               PersonalAccessTokens.validate_and_introspect(
+                 Faker.UUID.v4(),
+                 FakerRandom.random_between(3, 100)
+               )
+
+      # non existent PAT for a user
```

Comment:
question: 
why not individual tests?

Reply:
Because they were just detailed different reasons of the main use case 🙈 

Anyway I changed to this fa4a3b66b0a539833c64fbdd14fc31d645fa465a. Let me know if that's ok.